controllers/posts.js

```diff
@@ -0,0 +1,55 @@
+const Post = require("../model/posts");
+const Like = require("../model/likes");
+const Comment = require("../model/comments");
+
+const PostsController = {
+  async Index(req, res) {
+    try {
+      const { session } = req;
+      const post = new Post();
+      const posts = await post.getPosts();
```

Comment:
Surely getPosts should be a static method of the post class, this way you don't need to instantiate an empty post just to get all the posts?

Reply:
Check the comments below in the model class: https://github.com/makersacademy/instagram-challenge/pull/878#discussion_r750518059